### PR TITLE
Bump to v3.4.2

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by mheon** [comment]
Lot of small changes this time around - lot of commits but most are docs.

**Comment by rhatdan** [comment]
LGTM
If it passes tests.

**Comment by TomSweeneyRedHat** [comment]
@mheon, couple red tests

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -389,7 +408,7 @@
 - Fixed a bug where images with empty layers were stored incorrectly, causing them to be unable to be pushed or saved.
 - Fixed a bug where the `podman rmi` command could fail to remove corrupt images from storage.
 - Fixed a bug where the remote Podman client's `podman save` command did not support the `oci-dir` and `docker-dir` formats ([#9742](https://github.com/containers/podman/issues/9742)).
-- Fixed a bug where volume mounts from `podman play kube` created with a trailing `/` in the container path were were not properly superceding named volumes from the image ([#9618](https://github.com/containers/podman/issues/9618)).
+- Fixed a bug where volume mounts from `podman play kube` created with a trailing `/` in the container path were were not properly superseding named volumes from the image ([#9618](https://github.com/containers/podman/issues/9618)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:411]:
```suggestion
- Fixed a bug where volume mounts from `podman play kube` created with a trailing `/` in the container path were were not properly superseding named volumes from the image ([#9618](https://github.com/containers/podman/issues/9618)).
```

**Comment by TomSweeneyRedHat** [comment]
one nit and a whole lot of red unhappiness in the tests

File: RELEASE_NOTES.md (hunk 1 above)
Comment (rhatdan) [reply RELEASE_NOTES.md:411]: The above fix has a "were were" in it.

Comment (mheon) [reply RELEASE_NOTES.md:411]: These are old release notes, so I don't want to change them beyond spelling fixes

Comment (rhatdan) [reply RELEASE_NOTES.md:411]: Sure, get the tests to pass and we can merge.

Comment (TomSweeneyRedHat) [reply RELEASE_NOTES.md:411]: Thanks on the were were @rhatdan, thought I'd killed one in my comment.

**Comment by mheon** [comment]
We have a request to get that fix landed, so that could be problematic.

**Comment by Luap99** [comment]
I am able to get the test working with this diff:
```diff
diff --git a/libpod/container_log_linux.go b/libpod/container_log_linux.go
index 4029d0af7..b6b780bab 100644
--- a/libpod/container_log_linux.go
+++ b/libpod/container_log_linux.go
@@ -140,6 +140,7 @@ func (c *Container) readFromJournal(ctx context.Context, options *logs.LogOption
                        doTail = false
                }
                lastReadCursor := ""
+               partial := ""
                for {
                        select {
                        case <-ctx.Done():
@@ -229,6 +230,12 @@ func (c *Container) readFromJournal(ctx context.Context, options *logs.LogOption
                                logrus.Errorf("Failed parse log line: %v", err)
                                return
                        }
+                       if logLine.Partial() {
+                               partial += logLine.Msg
+                               continue
+                       }
+                       logLine.Msg = partial + logLine.Msg
+                       partial = ""
                        if doTail {
                                tailQueue = append(tailQueue, logLine)
                                continue
```
 This is not a breaking change, it just combines the partial lines into one single line like it is done for the file backend:
https://github.com/containers/podman/blob/92da8682b3977a6ef57a6f73dde2f2aef6186d19/libpod/container_log.go#L91-L97

**Comment by mheon** [comment]
Force-pushed with fix - thanks bunches @Luap99

**Comment by mheon** [comment]
Looks good, integration tests are all passing.

**Comment by mheon** [comment]
System tests are failing, unrelated to earlier failures. I think I might have grabbed a patch from Ed that depends on functionality in `system connection ls` that isn't in this branch.

**Comment by mheon** [comment]
Pulled the patch out, repushed

**Pull request merged**